release-notes/releases-index.json

```diff
@@ -0,0 +1,44 @@
+{
+    "core-releases-index": [
+        {
+            "channel-version": "2.1",
+            "latest": "2.1.4",
+            "release-date":"2018-09-11",
+            "lts": true,
+            "eol": false,
+            "eol-date": null,
+            "releases.json": "https://github.com/dotnet/core/blob/master/release-notes/2.1/releases.json",
+            "lifecycle-policy": "https://www.microsoft.com/net/support/policy"
+        },
+        {
+            "channel-version": "2.0",
+            "latest": "2.0.9",
+            "release-date":"2018-05-21",
```

Comment:
I get these three property names. I'm wondering if we can do better. How about (in order):

- version
- latest-release
- latest-release-date

Reply:
latest-release and latest-date make sense. channel-version has a specific meaning in both files that I would like to retain.